agent/rpm/Makefile

```diff
@@ -53,8 +53,7 @@ patches:
 .PHONY: tarball
 tarball:
 	mkdir -p ${RPMSRC}
-	git submodule init
-	git submodule update
+	(cd ${AGENT}/../; git submodule init; git submodule update)
```

Comment:
Why does it matter? You can do `git submodule init/update` from anywhere in the tree.

Reply:
Apparently not on a RHEL 7 box ...

Reply:
I am skeptical: I'll go test.